### PR TITLE
Add PodSecurityPolicy for sysdig-agent

### PR DESCRIPTION
**Pull request by gregswift**

### CODE REVIEW

File: agent_deploy/kubernetes/sysdig-agent-configmap.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ data:
     configmap: true
     ### Agent tags
     # tags: linux:ubuntu,dept:dev,local:nyc
+    tags: region:use,facility:ewr1,tier:int,cloud:packet
```
Comment (mbreitung) [inline agent_deploy/kubernetes/sysdig-agent-configmap.yaml]: did you mean to create this PR with your internal tags?

**Comment by gregswift** [comment]
Nope. Been meaning to fix thay

File: agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml [hunk 2]
```diff
@@ -61,3 +61,11 @@ rules:
...
   - list
   - watch
+- apiGroups:
+  - extensions
+  resourceNames:
+  - sysdig-agent
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - use
```
Comment (balinaveen) [inline agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml]: The podsecuritypolicy settings seem reasonable to me. but to ensure that the policy is consciously enabled, can  the "verbs: use" setting be commented out?

Comment (gregswift) [reply agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml]: Nope. I don't believe PSP works without that

**Review by mbreitung — APPROVED** [review]

**Review by nathan-b — APPROVED** [review]
Approving in ignorance.

**Comment by balinaveen** [comment]
Given that PSP is now deprecated (see https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/) I don't think we need this.

**Comment by nathan-b** [comment]
In that case, any objection to me closing it so GitHub doesn't nag me about it anymore?

**Comment by balinaveen** [comment]
@bryanseay  can we reject this PR ad close it since k8s PSP is deprecated and will be going away?

**Comment by gregswift** [comment]
:shrug: There ya go :D feel free to re-open... we manage it on our own already

**Comment by nathan-b** [comment]
Thanks Greg :)

**Pull request closed without merging**